Asterisk.2013/Asterisk.NET/Manager/Event/MusicOnHoldEvent.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace AsterNET.Manager.Event
+{
+    /// <summary>
+    ///     The MusicOnHoldEvent event triggers when the music starts or ends playing the hold music.<br />
+    ///     See <see target="_blank"  href="LINK">LINK</see>
```

Comment:
Why is this link blank?

Reply:
forgot to fill it in
will fix

Reply:
I can't really find docs for that event
I removed it for start/stop events